physionet-django/console/views.py

```diff
@@ -1550,6 +1550,22 @@ def training_list(request, status):
 
     display_training = training_by_status[status]
 
+    if request.method == 'POST':
+        if "search" in request.POST:
+            display_training = search_training_applications(request, display_training)
+            if status == 'review':
```

Comment:
It's a matter of subjective preference but to me this code would look better like this:
```python
template_by_status = {
  'review': 'console/review_training_table.html',
  'valid': 'console/valid_training_table.html',
  'expired': 'console/expired_training_table.html',
  'rejected': 'console/rejected_training_table.html',
}
return render(request, template_by_status[status], {'trainings': display_training, 'status': status})
```

No branching and it's immediately clear that the template is the only thing that's different for different statuses. It's also possible to go as far as to do `f"console/{status}_training_table.html"` but coupling the file name doesn't feel right.